string/strcspn.c

```diff
@@ -27,46 +27,65 @@
 
 /* Return the length of the maximum initial segment of S
    which contains no characters from REJECT.  */
-size_t
-STRCSPN (const char *str, const char *reject)
+size_t STRCSPN (const char *str, const char *reject)
```

Comment:
Just a coding convention issue:
This goes against the convention found within the library. Though inconsistent, most variable types are declared on the previous line, such as the previous declaration of `STRCSPN`.